src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo.cs

```diff
@@ -215,7 +231,7 @@ private int BinarySearch(string name)
             return -1;
         }
 
-        #region Construction
+#region Construction
```

Comment:
This is whatever the formatter did.  I don't touch these :)   I also don't generally like or use regions in the first place.